tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/config_CBMC_PROOF_ASSUMPTION_DOES_NOT_HOLD/OutputARPRequest_harness.c

```diff
@@ -0,0 +1,47 @@
+/* FreeRTOS includes. */
```

Comment:
I thought we're copying the harness.c, cbmc-batch.yaml, and other files into the subdirectories generated by your configuration script? If so, please remove this file from the PR